### PR TITLE
Dublication selector for class header-top removed from global.js

### PR DESCRIPTION
**Pull request by codersantosh**
wp.org username: codersantosh

### CODE REVIEW

File: inc/template-tags.php [hunk 1]
```diff
@@ -114,7 +114,7 @@ function twentyseventeen_edit_link( $id ) {
 	} elseif ( get_post( $id ) ) {
 		$type = __( 'Post', 'twentyseventeen' );
 	}
-	$link = edit_post_link(
```
Comment (josephfusco) [inline inc/template-tags.php:117]: This should be made into a separate Pull Request

Comment (codersantosh) [reply inc/template-tags.php:117]: Thanks, will do that.

Comment (codersantosh) [reply inc/template-tags.php:117]: I tried to create new pull request by creating branch but failed. If you can take care of this pull request, may be I am able to create a new pull request.

Comment (josephfusco) [reply inc/template-tags.php:117]: @codersantosh Just a tip, avoid putting changes in the **master** branch of your fork. Create a branch off of master for each separate pull request. This will help break up edits and make for easier merges. :)

Comment (codersantosh) [reply inc/template-tags.php:117]: @josephfusco Thank you so much for your tip, I tried to do that, but the both commits appear in pull request so I closed the pull request (  https://github.com/WordPress/twentyseventeen/pull/104 ), I am not so good at git :( .
Anyway I believe it is mistake in the function twentyseventeen_edit_link ( https://github.com/WordPress/twentyseventeen/blob/master/inc/template-tags.php#L116 ), Also I think it is just duplicate function of twentyseventeen_edit_post_link ( https://github.com/WordPress/twentyseventeen/blob/master/inc/template-tags.php#L58 ). But the second function ( twentyseventeen_edit_post_link ) is using by the templates too.

Comment (josephfusco) [reply inc/template-tags.php:117]: @codersantosh this pull request looks good now. Some of the code has changed so those links https://github.com/WordPress/twentyseventeen/blob/master/inc/template-tags.php#L116 & https://github.com/WordPress/twentyseventeen/blob/master/inc/template-tags.php#L58 are no longer showing accurate line numbers.

Fee free to DM me on the Make WordPress Slack team if you need assistance with making another pull request. My slack username is **`@joefusco`**

**Review by josephfusco — APPROVED** [review]

**Comment by josephfusco** [comment]
~~@davidakennedy all set here :)~~

File: inc/template-tags.php (hunk 1 above)
Comment (codersantosh) [reply inc/template-tags.php:117]: Thanks @josephfusco

**Comment by josephfusco** [comment]
@codersantosh Unfortunately global.js has changed a lot since #89 was merged, implementing the new design, and I am no longer seeing the issue.

We are probably good to close this issue. Since there is a lot of new code in the theme now, feel free to submit more changes if you see anything else. 👍 Thank you for taking the time to submit this pull request though :)
